src/browser/atom-application.coffee

```diff
@@ -411,6 +415,41 @@ class AtomApplication
         console.log("Killing process #{pid} failed: #{error.code ? error.message}")
     delete @pidsToOpenWindows[pid]
 
+  saveState: ->
+    states = []
+    for window in @windows
+      if loadSettings = window.getLoadSettings()
+        states.push(_.pick(loadSettings,
+          'initialPaths'
+          'isSpec'
```

Comment:
I was planning on doing that, but I think I should actually just filter out the spec windows for now. What do you think of that?